### ISSUE TITLE
New error:   no checking of IP address

### ISSUE THREAD

**Issue by tthtlc** [gyoisamurai/GyoiThon#42]
First a scan was started.  (using -p for exploit option).
It was scanning all the time and partial output here:

[*] 1102/1219 exploit:windows/misc/bigant_server_usv, targets:1
[*] 1103/1219 exploit:windows/misc/bopup_comm, targets:1
[*] 1104/1219 exploit:windows/misc/citrix_streamprocess, targets:1
[*] 1105/1219 exploit:windows/misc/cloudme_sync, targets:1
[*] 1106/1219 exploit:windows/misc/commvault_cmd_exec, targets:1
[*] 1107/1219 exploit:windows/misc/disk_savvy_adm, targets:1
[*] 1108/1219 exploit:windows/misc/hp_dataprotector_cmd_exec, targets:1
[*] 1109/1219 exploit:windows/misc/hp_dataprotector_exec_bar, targets:2
[*] 1110/1219 exploit:windows/misc/hp_dataprotector_install_service, targets:1
[*] 1111/1219 exploit:windows/misc/hp_dataprotector_traversal, targets:1
[*] 1112/1219 exploit:windows/misc/hp_imc_dbman_restartdb_unauth_rce, targets:1
[*] 1113/1219 exploit:windows/misc/hp_imc_dbman_restoredbase_unauth_rce, targets:1
[*] 1114/1219 exploit:windows/misc/hp_loadrunner_magentproc_cmdexec, targets:1
[*] 1115/1219 exploit:windows/misc/hp_omniinet_1, targets:5
[*] 1116/1219 exploit:windows/misc/hp_omniinet_2, targets:5
[*] 1117/1219 exploit:windows/misc/hp_omniinet_3, targets:1
[*] 1118/1219 exploit:windows/misc/hp_omniinet_4, targets:1
[*] 1119/1219 exploit:windows/misc/ib_isc_attach_database, targets:11
[*] 1120/1219 exploit:windows/misc/ib_isc_create_database, targets:11

but yet, no error indicated.   Next check with netstat, ifconfig and ping, the target address is no longer running.   
In summary, when the target IP is no longer existing, Gyoithon will continue scanning and running as if the IP is still there.

Can you confirm this issue?

THanks.
